### PR TITLE
WIP: Classifier: Add PreviousAnnotations

### PR DESCRIPTION
**Pull request by eatyourgreens**
Add a PreviousAnnotations model, adapted from Anti-Slavery Manuscripts.
The model requests reductions from Caesar for a given workflow, subject and reducer key, then further reduces them to an array of consensus annotation models for a given frame.

Tests are currently set up with a couple of existing ASM subjects. One has reductions for frame 1 and frame 2. The other has no reductions.

TODO:
- [ ] mock up some Caesar responses.
- [ ] write some tests.
- [ ] connect the `PreviousAnnotations` model to the `Subject` model.

Package:
lib-classifier

Towards #1271.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import asyncStates from '@zooniverse/async-states'
+import { GraphQLClient } from 'graphql-request'
+import { flow, types } from 'mobx-state-tree'
+
+export const caesarClient = new GraphQLClient('https://caesar.zooniverse.org/graphql')
+const CONSENSUS_SCORE_TO_RETIRE = 3
+const MINIMUM_VIEW_TO_RETIRE = 5
+
+const Annotation = types.model({
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js]: Is this name confusing, given that we already have an `Annotation` model for classifications?

Comment (srallen) [reply packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js]: Yes. Could we call caesar reduction related models for transcriptions `TranscriptionReductions` and `TranscriptionReduction` instead?

Comment (eatyourgreens) [reply packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js]: Sure. Are these reductions only run for transcriptions?

Comment (srallen) [reply packages/lib-classifier/src/store/PreviousAnnotations/PreviousAnnotations.js]: It'll only be for projects configured to use the transcription task and downstream with caesar to use the transcription specific reducers, so yes. We don't know how this full process would work with any other projects and task types and if the models will be the same.

**Pull request closed without merging**